invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/components/AvailableItems/AvailableItems.js

```diff
@@ -80,6 +105,15 @@ export default class AvailableItems extends Component {
     );
   }
 
+  _rowActionButton(row) {
+    const { loan } = this.props;
+    if (loan.metadata.state === 'PENDING') {
+      return this._checkoutItemButton(row, loan);
+    } else {
+      return this._assignItemButton(row);
```

Comment:
For me it is a bit unclear, in which case we assign the item? (I mean the else condition)

Reply:
we don't assign the item here, we render either the checkoutButton when the loan is `PENDING` or assign item button when the loan is any other state, to give the possibility to replace the item. I'm not sure how could I make it more descriptive, to leave a comment in here ?